stage #1/module #8. Concurrency/lesson #1. Concurrency/content.md

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
Will you create another pull request for some classes from the concurrency package?